astro/src/content/docs/_shared/_collected-metrics.mdx

```diff
@@ -0,0 +1,531 @@
+
+We will never collect anything personally identifiable about your users or customers. We may collect new metrics as we add new features, but those will be outlined in our release notes.
+
+Here's the breakdown of what FusionAuth deployments which this feature will collect data from, and how it will happen:
+
+-   Self-hosted: You get to choose whether or not you want to share your data with us. You can turn usage data collection off by API, setup wizard, or in the admin UI, under XXX. You can also block the outbound request at the network--just block the hostname https://usage-stats.fusionauth.io. If you decide to share usage data, we will protect it.
+-   By default, any upgraded instances will NOT send usage data.
+-   By default, any new instances will send usage data, but you can opt out at instance creation or any time after.
+-   For those who choose to have us host your FusionAuth deployment in FusionAuth Cloud, data collection can not be disabled. This data helps us provide top-notch service and security.
+- If you are on the Community plan, which all unlicensed versions of FusionAuth are, and enable this feature, we collect data but won't tie it to your identity. 
+- If you are on the Licensed Community plan (available in version 1.52.0 and later, which enables passkeys and other features), we tie usage data to your account and identity.
+-   Other plans: We tie the data to your account and identity via the `instanceId` field.
+
+If you have questions or concerns, we're here to chat. Drop us an email at [devrel@fusionauth.io](mailto:devrel@fusionauth.io), or stop by the [FusionAuth Community forums](/community/forum/).
+
+## Frequently Asked Questions
+
+
+### How Will This Impact Performance? 
+
+We have been developing this feature internally for quite some time. First, stats collection is intentionally throttled to not use additional resources. We then chose to only have the queries run once per day. We load tested with a variety of customer sizes and database configurations until we were able to collect this data without the collection impacting deployment performance. As a further failsafe, we have built in functionality that lets us dynamically disable any stat collection that does cause a problem, without the need to release or upgrade the instance.
+
+### How Does FusionAuth Ensure that My Customer Data Remains Private?
+
+Your privacy, and that of your customers, is of the utmost importance to us. We have designed this system so that it only collects aggregated data, so that we do not see data on any one specific instance. Further, we do not collect any data that might reveal any information about your customers, such as webhook or OIDC URLs.
+
+### Who Owns the Data that You Collect? Who Decides How that Data is Used?
+
+The FusionAuth [license agreement](/license) covers the specifics of how we collect and use data. For more detailed information, please see the following sections:
+
+-   Section 2.1.6 covers self-hosting
```

Comment:
I don't see how 2.1.6 covers self-hosting? That talks about auditing?

Reply:
I have no idea. In fact, I don't see anything in the licensing that covers how we use data collected from self-hosted instances.